mm2src/coins/lp_coins.rs

```diff
@@ -4300,6 +4300,8 @@ impl CoinsContext {
 }
 
 /// This enum is used in coin activation requests.
+/// TODO: should we use #[serde(tag = "type", content = "params")] for this PrivKeyActivationPolicy like for the Eth policy,
+/// to have them identical in activation requests
```

Comment:
There is an issue for this https://github.com/KomodoPlatform/komodo-defi-framework/issues/2522

Reply:
Maybe fix this in a separate PR
(will be a breaking change for the API)